pkgs/development/tools/rust/cargo-lambda/default.nix

```diff
@@ -9,7 +9,7 @@
 , stdenv
 , CoreServices
 , Security
-, zig
```

Comment:
It is better to set this on top level, as I am doing on https://github.com/NixOS/nixpkgs/pull/307268

Reply:
Ah thank you, I was just about to ask. Should I change this for all packages then?

Reply:
Hmm or maybe it's better to just merge your PR first.

Reply:
I believe some updates are in order.

1. In the documentation, use `zig` instead of `zig_VERSION`
2. In the examples you are touching (namely, those that work with zig 0.12), do like #307268
  - Optionally migrate them to by-name 
  - No worries about merge conflicts on my side, I can deal with them easily (it's just removing the offending commit)